plugin.video.piped/lib/services/entrypoint.py

```diff
@@ -0,0 +1,28 @@
+from threading import Thread
+import xbmc
+
+from lib.services.httpserver import HttpService
+from lib.services.scheduler import Scheduler
+from lib.services.player import Player
+
+class Service(xbmc.Monitor):
+	def __init__(self):
+		httpservice = Thread(target=HttpService)
+		scheduler = Scheduler()
+		httpservice.daemon = True
+		scheduler.daemon = True
+		player = Player()
+		player.isPlaying()
+
+		while not self.abortRequested():
```

Comment:
If `abortRequested()` results in `True`, the `httpservice` and `scheduler` aren't stopped properly and could cause Kodi to wait for a long time.